### ISSUE TITLE
CIDER version

### ISSUE THREAD

**Issue by am2605** [purcell/emacs.d#386]
Hello,

Are you able to advise how I can "pin" cider to the stable version please?  

I tried putting (add-to-list 'package-pinned-packages '(cider . "melpa-stable") t) in my init-local.el but   I seem to have 20161007.924 installed but when I connect to a clojure repl started with lean repl I get the warning:

WARNING: CIDER's version (0.14.0-snapshot) does not match cider-nrepl's version (0.13.0). Things will break!

Thanks!
Andrew.

**Comment by purcell** [comment]
I _think_ `package-pinned-packages` only affects installation, so you'll have to uninstall/delete the unwanted snapshot package, and reinstall the package so that you get the stable version instead.

**Comment by am2605** [comment]
Thanks.  I also needed to add the melpa-stable repo, and now I've got it sorted out.

Thanks for your help, and this great project as always 



On Sat, Oct 8, 2016 at 4:50 PM +1100, "Steve Purcell" <notifications@github.com> wrote:












I think package-pinned-packages only affects installation, so you'll have to uninstall/delete the unwanted snapshot package, and reinstall the package so that you get the stable version instead.
